src/ios/CDVInsert.m

```diff
@@ -73,10 +64,37 @@ - (void)initSDK:(CDVInvokedUrlCommand *)command {
                             [notifyCenter removeObserver:errorToken];
                             [self sendPluginResult:CDVCommandStatus_ERROR command:command message:@"Insert.IO init error. No further information was provided by the SDK :("];
                         }];
-    [[InsertManager sharedManager] initSDK:appKey companyName:companyName];
+    NSDictionary *userAttributes;
+    NSString *visitorId;
+    NSString *accountId;
+    if (command.arguments.count >= 3) {
+        userAttributes = command.arguments[0];
+        visitorId = command.arguments[1];
+        accountId = command.arguments[2];
+    } else {
+        [self sendPluginResult:CDVCommandStatus_OK command:command message:@"Expecting 2 arguments: actionName, params"];
+    }
+    InsertInitParams *initParams = [[InsertInitParams alloc] init];
+    if (userAttributes)
```

Comment:
Can you add braces to single if statements (just for the sake of conventions)

Reply:
Done.